android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -598,6 +597,14 @@ private void doMenuAction(int menuItemId) {
                 }
                 break;
         }
+
+        if(menuItemId != R.id.nav_schedule
```

Comment:
I got your point. Sorry for not realizing it. Your implementation of per case setting was better in case there are so many conditions